highfidelity/hifi
# moved checkEditPermissionsAndUpdate function to higher scope

This is a fix for https://highfidelity.fogbugz.com/f/cases/14671/Error-in-edit-js-when-restarting-that-script

TEST PLAN

- Start Hifi dev build and open the log.
- Reload scripts (Ctrl + R) and search logs for "checkEditPermissionsAndUpdate". This line should not appear in the log at all.
- Open create menu and click on a few random items, then close create menu and check the log again. The line, "checkEditPermissionsAndUpdate", should still not appear in the log
